torq.sh

```diff
@@ -69,7 +68,7 @@ summary() {
     printf "%-8s | %-14s | %-6s |\n" `date '+%H:%M:%S'` "$1" "down"                                 # summary table row for down process
   else
     pid=$(findproc $1)
-    port=`netstat -pl 2>/dev/null | grep $pid | awk '{ print $4 }' | head -1 | cut -c 3-`           # get port process is running on 
+    port=`netstat -nlp 2>/dev/null | grep $pid | awk '{ print $4 }' | head -1 | awk -F: '{ print $2 }'`  
```

Comment:
Can you put the comment back on this line? it'll have to be slightly out of line with the others given this is a longer line, but that's better than no comment